docs/Algorithms.md

```diff
@@ -1,8 +1,35 @@
 # Implemented Compression Methods
 
-Each compression method receives its own hyperparameters that are organized as a dictionary and basically stored in a JSON file that is deserialized when the training starts. Compression methods can be applied separately or together producing sparse, quantized, or both sparse and quantized models. For more information about the configuration, refer to the samples.
+## Post-training Compression
 
-- [Quantization](./compression_algorithms/Quantization.md)
+- [Quantization](./usage/post_training_compression/post_training_quantization/Usage.md) (OpenVino, PyTorch, ONNX, TensorFlow)
```

Comment:
Maybe it's better to call it as Post Training Quantization (PTQ).
@alexsu52 what do you think?

Reply:
I agree with you.

Reply:
Done